### PR TITLE
added openFrameworks 0.9.3 docset

### PR DESCRIPTION
**Pull request by icq4ever**
in docset.json, just update new docset author info. and update versions.
older version's author informations are written in readme.md

### CODE REVIEW

File: docsets/openFrameworks/docset.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         },
         {
             "version": "0.8.4",
-            "archive": "versions/0.8.4/Sample.tgz",
+            "archive": "versions/0.8.4/openFrameworks.tgz",
```
Comment (Kapeli) [inline docsets/openFrameworks/docset.json]: You need to add this archive (`versions/0.8.4/openFrameworks.tgz`) to the repo, or remove the `specific_version` entry completely.

**Comment by Kapeli** [comment]
@marcusficner Is it ok with you if @icq4ever takes over the maintenance of this docset?

File: docsets/openFrameworks/docset.json (hunk 1 above)
Comment (icq4ever) [inline docsets/openFrameworks/docset.json]: I'll upload old openFrameworks.tgz.

**Comment by icq4ever** [comment]
what's this ? do I have to close this PR and create a new PR?

**Comment by Kapeli** [comment]
No. The `0.8.4` docset you added contains a lot of entries that point to missing files.

Why not just use the old 0.8.4 docset? You can grab it from https://kapeli.com/feeds/zzz/user_contributed/build/openFrameworks/openFrameworks.tgz

**Comment by icq4ever** [comment]
thanks. I replace with that link.

**Comment by Kapeli** [comment]
Great. I'll go ahead and merge this. Thank you! :+1: 

@marcusficner if you're not ok with any of the changes, let me know and I'll revert them.

**Comment by marcusficner** [comment]
@Kapeli Looks good to me :+1:  No problem, @icq4ever can take over. Thanks!

**Pull request merged**